src/rules/quotemarkRule.ts

```diff
@@ -338,6 +354,20 @@ function isTypeCheckWithOldTsc(node: ts.StringLiteral | ts.NoSubstitutionTemplat
     return node.parent.getChildren().some(n => n.kind === ts.SyntaxKind.TypeOfExpression);
 }
 
+function containsOctalEscapeSequence(node: StringLiteralLike, sourceFile: ts.SourceFile) {
+    // Octal sequences can go from 1-377 (255 in octal), but let's match the prefix, which will at least be \1-\77
+    return /\\[1-7][0-7]?/.test(node.getText(sourceFile));
```

Comment:
If the node is the actual `ts.StringLiteral` type, this should go off `node.text`, right? IIRC that's a bit more performant than getting text from the source file.

It's unlikely this would be an actual performance concern given the context this is used in, but I'd really like to discourage folks from using `.getText` whenever possible.

Reply:
`getText` actually did not include the literal backslashes here, for some reason. Since I needed to match the backslash, I had to get the text from the file directly.